server/handlers.go

```diff
@@ -180,11 +180,14 @@ func (svr *Server) handlePostShared(w http.ResponseWriter, r *http.Request, comm
 			Err:  fmt.Errorf("put request failed with commitment %v (commitment mode %v): %w", comm, meta.Mode, err),
 			Meta: meta,
 		}
-		if errors.Is(err, store.ErrEigenDAOversizedBlob) || errors.Is(err, store.ErrProxyOversizedBlob) {
-			// we add here any error that should be returned as a 400 instead of a 500.
-			// currently only includes oversized blob requests
+		var errAPI api.ErrorAPI
+		switch {
+		case is400(err):
 			http.Error(w, err.Error(), http.StatusBadRequest)
-		} else {
+		case errors.As(err, &errAPI) && errAPI.ErrorCode() == http.StatusServiceUnavailable:
+			// this tells the caller (batcher) to failover to ethda b/c eigenda is temporarily down
+			http.Error(w, err.Error(), http.StatusServiceUnavailable)
+		default:
```

Comment:
I thought you wanted to add 201

Reply:
That's a whole separate thing. Will add the 201 when we fix https://github.com/Layr-Labs/eigenda-proxy/issues/173